proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -404,3 +404,26 @@ enum Infraction {
 message ValidatorUpdates {
   repeated tendermint.abci.ValidatorUpdate updates = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
 }
+
+// ConsPubKeyRotationHistory contains a validator's consensus public key rotation history.
+message ConsPubKeyRotationHistory {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // operator_address defines the address of the validator's operator; bech encoded in JSON.
+  string operator_address = 1 [(gogoproto.moretags) = "yaml:\"operator_address\""];
+  // old_cons_pubkey is the old consensus public key of the validator, as a Protobuf Any.
+  google.protobuf.Any old_cons_pubkey = 2 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey"];
```

Comment:
```suggestion
  google.protobuf.Any old_cons_pubkey = 2;
```

is this needed?

Reply:
Yes, this is to keep track of the validator rotated addresses. we are using the history object `ConsPubKeyRotationHistory`

Reply:
sorry meant the proto annotations

Reply:
not sure though, I followed [this](https://github.com/cosmos/cosmos-sdk/blob/202485023cce681d38666b0d7a85f1100c77d2d9/proto/cosmos/staking/v1beta1/tx.proto#L76), we are using the same proto annotation to pubkey while creating validator.

Reply:
we can leave for now, but would be nice to better understand this annotation